src/main/java/IsaacCharacterGeneratorApplication.java

```diff
@@ -1,15 +1,39 @@
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Random;
+import javax.sound.sampled.Line;
 
 public class IsaacCharacterGeneratorApplication {
     public static void main(String[] args) {
-        IsaacCharactersRepository isaacCharactersRepository = new IsaacCharactersRepository();
-        List<String> issacCharacters = isaacCharactersRepository.getCharacters();
+//        IsaacCharactersRepository isaacCharactersRepository = new IsaacCharactersRepository();
+//        List<String> issacCharacters = isaacCharactersRepository.getCharacters();
 //        RandomCharacterPicker randomCharacterPicker = new RandomCharacterPicker();
 //        String character = randomCharacterPicker.pick(issacCharacters);
+
+        gettingUserInput();
+
+    }
+
+    private static void gettingUserInput() {
+
+        CollectingCharactersDataViaTerminal CollectingCharactersDataViaTerminal = new CollectingCharactersDataViaTerminal();
+        System.out.println("Type character names from which you want to roll:");
+
+        while (true) {
```

Comment:
Tego na tę chwilę nie spradzam bo zmieni to się jak wprowadzisz zmiany do `CollectingCharactersDataViaTerminal`